### PR TITLE
Feature/display coffee shops

### PR DESCRIPTION
**Pull request by aistrate**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.CoffeeShopsView__container {
+  margin: 20px 6vw 0 6vw;
+}
+
+.CoffeeShopsView__error {
+  color: red;
+}
```
Comment (adrianbunea) [inline src/App.css]: I'm curious what naming convention is this? Do you follow any?

**Review by adrianbunea — APPROVED** [review]

**Comment by adrianbunea** [comment]
This looks great! Maybe it is just a personal preference but I think the dots could be a lot larger considering the low density of points.

File: src/App.css (hunk 1 above)
Comment (aistrate) [reply src/App.css]: I follow the BEM naming convention.

I see now that CoffeeShopsView__error could have been just 'error', as it's a reusable class.

**Review by dasilvaluis — APPROVED** [review]
Clean work!

File: src/App.css (hunk 1 above)
Comment (adrianbunea) [reply src/App.css]: You can keep it for now as a part of the `CoffeeShopsView`, until you actually need it to be reused. My question arose from the letter casing you were using, I was familiar with BEM but is `PascalCase` normal with BEM?

Comment (aistrate) [reply src/App.css]: I was thinking the "Block" part of BEM is in this case the React component (which is capitalized). I will change it to lower case.

Comment (aistrate) [reply src/App.css]: Actually, I'll change it to coffee-shops-view__container.

**Pull request merged**